source/_integrations/ollama.markdown

```diff
@@ -44,4 +44,7 @@ Max history messages:
 Keep alive:
   description: Duration in seconds for the Ollama host to keep the model in memory after receiving a message (-1 = no limit, 0 = no retention). Default value is -1.
 
+Context tokens:
+  description: Number of context tokens to use in conversation with the model.  Default value is 2048 — using this value ensures that the default context token size of the model is used.
```

Comment:
What do you mean by "using this value"? 2048? Does it mean I have to enter 2048, although it is the default?

Reply:
I did not know how to make a default empty value, only 2048 should be suggested as the default.

If someone could guide me into making sure that voluptuous accepts an empty value as default without adding 2048 in the code, I would gladly do it.

Reply:
oh, I was not trying to tell you what to do. Just trying to make sure I understand the steps correctly, from a language perspective. So, if the user wants does not have to enter anything manually if they are happy with the default, maybe we could rephrase to .".. value is 2048. This value ensures..."?